Snippets/ASQ/ASQN_10/Usage.cs

```diff
@@ -15,7 +15,19 @@ void UseTransport(EndpointConfiguration endpointConfiguration)
     {
         #region AzureStorageQueueTransportWithAzure
 
+        var connectionString = "DefaultEndpointsProtocol=https;AccountName=[ACCOUNT];AccountKey=[KEY];";
+
         var transport = endpointConfiguration.UseTransport<AzureStorageQueueTransport>();
+        transport.UseQueueServiceClient(new QueueServiceClient(connectionString));
+
+        //Table and Blob clients don't need to be specified if native delayed delivery is disabled
+        //transport.DelayedDelivery().DisableDelayedDelivery();
```

Comment:
I think we still need table for native pub-sub (which there is not an API to disable)